src/test/java/teammates/storage/sqlapi/FeedbackSessionsDbTest.java

```diff
@@ -0,0 +1,255 @@
+        feedbackSession.setName("");
+
+        assertThrows(InvalidParametersException.class, () -> feedbackSessionsDb.createFeedbackSession(feedbackSession));
+    }
+
+    @Test
+    public void testCreateSession_nullParams_throwsAssertionError()
+            throws InvalidParametersException, EntityAlreadyExistsException {
+        assertThrows(AssertionError.class, () -> feedbackSessionsDb.createFeedbackSession(null));
+    }
+
+    @Test
+    public void testGetFeedbackSession_sessionExists_success() {
+        FeedbackSession feedbackSession = createnewSession();
+        UUID uuid = feedbackSession.getId();
+        mockHibernateUtil.when(() -> HibernateUtil.get(FeedbackSession.class, uuid)).thenReturn(feedbackSession);
+
+        FeedbackSession sessionFetched = feedbackSessionsDb.getFeedbackSession(uuid);
+
+        mockHibernateUtil.verify(() -> HibernateUtil.get(FeedbackSession.class, uuid), times(1));
+        assertEquals(feedbackSession, sessionFetched);
+    }
+
+    @Test
+    public void testGetFeedbackSession_sessionDoesNotExists_returnNull() {
+        UUID randomUuid = UUID.randomUUID();
+        mockHibernateUtil.when(() -> HibernateUtil.get(FeedbackSession.class, randomUuid)).thenReturn(null);
+
+        FeedbackSession sessionFetched = feedbackSessionsDb.getFeedbackSession(randomUuid);
+
+        mockHibernateUtil.verify(() -> HibernateUtil.get(FeedbackSession.class, randomUuid), times(1));
+        assertNull(sessionFetched);
+    }
+
+    @Test
+    public void testUpdateFeedbackSession_success() throws InvalidParametersException, EntityDoesNotExistException {
+        FeedbackSession feedbackSession = createnewSession();
+        doReturn(feedbackSession).when(feedbackSessionsDb).getFeedbackSession(any(UUID.class));
+
+        feedbackSessionsDb.updateFeedbackSession(feedbackSession);
+
+        mockHibernateUtil.verify(() -> HibernateUtil.merge(feedbackSession));
+    }
+
+    @Test
+    public void testUpdateFeedbackSession_sessionDoesNotExist_throwsEntityDoesNotExistException()
+            throws InvalidParametersException, EntityDoesNotExistException {
+        FeedbackSession feedbackSession = createnewSession();
+        UUID uuid = feedbackSession.getId();
+        doReturn(null).when(feedbackSessionsDb).getFeedbackSession(uuid);
+
+        assertThrows(EntityDoesNotExistException.class,
+                () -> feedbackSessionsDb.updateFeedbackSession(feedbackSession));
+        mockHibernateUtil.verify(() -> HibernateUtil.merge(feedbackSession), never());
+    }
+
+    @Test
+    public void testUpdateFeedbackSession_sessionInvalid_throwsInvalidParametersException()
+            throws InvalidParametersException, EntityDoesNotExistException {
+        FeedbackSession feedbackSession = createnewSession();
+        UUID uuid = feedbackSession.getId();
+        feedbackSession.setName("");
+        doReturn(null).when(feedbackSessionsDb).getFeedbackSession(uuid);
```

Comment:
while the test might be passing, it is because the check for validity is happening first, we should return the feedbackSession instead of null here

Reply:
good catch! I have made the changes